### ISSUE TITLE
Идея: краудсорсинговый проект с актуальными отделами полиции по всей России (адреса, координаты, телефоны)

### ISSUE THREAD

**Issue by loalexandra** [developers-against-repressions/devs-against-the-machine#52]
Исходное [сообщение](https://t-do.ru/c/1497314240/1568)

> краудсорсинговый проект с актуальными отделами полиции по всей России (адреса, координаты, телефоны), который бы их отдавал по апи или скопом нуждающимся проектам. Есть открытые данные полиции и ещё некоторые источники, но в целом это мрак

**Comment by behlbor** [comment]
**Суть**: регулярно обновляемый датасет с открытым доступом (апи в общем-то необязательно)

**Зачем это нужно?** 
Такие проекты как Красная кнопка, Передачки и ОВД-Инфо сильно зависят от знания местоположения и телефонов тех мест, куда доставляют задержанных.

**В чем сложность?** 
1. Получение актуального списка контактов и адресов подразделений МВД. 2. Вычленение из них тех, куда могут привозить задержанных. 3. Поддержание актуальности 

**Существующие источники:**
https://mvd.ru/opendata, скорее всего конкретно этот безобразный и редко обновляемый [датасет](https://mvd.ru/opendata/7727739372-MVDOY).
https://mvd.ru/contacts/sites - региональные сайты полиции. Они разные, но вероятно, это по качеству и актуальности лучший источник - так как похоже, что они ведутся каждый своим регионом.
(Яндекс.Справочник)[https://yandex.ru/sprav/] или другие справочники организаций. Сложно предсказать, какова актуальность и неизвестно каким образом собираются данные.

**Почему краудсорсинг?**
Возможно, это единственный способ добиться качественного результата. А может, и нет - в любом случае начинать, если за это браться, стоит с углубленного ресёрча

**Comment by ololosha228** [comment]
@behlbor можете попробовать воспользоваться openstreetmap, 2gis, что бы были привязки телефонов к геоточкам (это важно, поскольку названия отделов могут меняться, адреса меняются, а вот геопозиция — нет)

**Comment by solapastor** [comment]
Копипаста из чата по хакатону
Ребзя, я мутаген до воскресенья. Простите пожалуйста, кто взялся за базу ОВД. Но в вс готов заниматься весь день. Пишите в личку. С меня база, апи, сервер (временный, но потом арендую постоянный за свои деньги). Нужен тот, кто шарит в парсинге HTML страниц и работал с апи справочников (openstreetmap, 2gis, yandex etc)
Tg: @solaanonymus

**Comment by tyomik-mnemonic** [comment]
за базу  osm по любому брать.  Могу помочь в части Развернуть pg+тюнинг , залить открытые данные + гис сервер и настройка его для рендеринга на клиенте . Воскресенье .

**Comment by solapastor** [comment]
Отлично. Напиши мне в тг, чтобы мы были на связи и в вс приступим

**Comment by antraksa** [comment]
Уже есть policemap.ru. Наверное лучше связаться с ними и предложить улучшения и помощь.

**Comment by MayneX** [comment]
@antraksa 

> «Карта полиции» реализована в партнёрстве с Объединённой группой общественного наблюдения и инициативой общественного мониторинга отделов полиции «Гражданин и полиция», **поддерживаемой руководством МВД.**

Завтра гумвд скажет, что это плохо, и проект исчезнет с серверов.

**Comment by antraksa** [comment]
Нет, не исчезнет. Карта не зависит от поддержки полиции. Я участвовала в разработке.

**Comment by tyomik-mnemonic** [comment]
Блин, Действительно , а чем по функциональности отличаться проект будет от policemap.ru?

**Comment by behlbor** [comment]
policemap.ru хороший проект, но у него другие, намного более глобальные цели и он идёт в другом направлении - сейчас это тщательный сбор (включая физическое хождение по отделам) очень подробных данных по крупным городам. 
А здесь речь о составлении очень конкретного простого датасета, но зато по всей России. Датасета который нужен различным приложениям.
